google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/RestoreTableRequest.java

```diff
@@ -29,37 +29,52 @@ public final class RestoreTableRequest {
   private final String sourceBackupId;
   private final String sourceClusterId;
   private final String sourceInstanceId;
+  private final String sourceProjectId;
 
   /**
    * Create a {@link RestoreTableRequest} object. It assumes the source backup locates in the same
-   * instance as the destination table. To restore a table from a backup in another instance, use
+   * instance and project as the destination table. To restore a table from a backup in another instance, use
```

Comment:
Add a comment:

"To restore a table from a backup in another project, use ..."